README.md

```diff
@@ -72,11 +72,18 @@ These ambitious startups seem very risky when they're just starting up. They nee
 We make angel investments in great founders who have found an important problem to solve. Then, we help set an ambitious vision and successfully create it.
 
 ## Our Investment Thesis
-First, we look for a startup that's found a group of Users with a important, painful, and valuable Problem. We call this having an "**Important Mission**."
```

Comment:
I liked these three sentences which summarize the idea in three simple concepts. The remainder (geography, etc.) is more detail focused.

That said, the section above `Our Strategy` covers this very well with "We make angel investment in *great founders* who have found an *important problem* to solve. Then, we help set an *ambitious vision* and successfully create it.

Perhaps we rename `Our Strategy` to `Our Investment Thesis` and change `Our Investment Thesis` to `Requirements` (and make requirements a sub-section?)?